pkg/gpu/tags/tags_nix.go

```diff
@@ -0,0 +1,39 @@
+// Unless explicitly stated otherwise all files in this repository are licensed
+// under the Apache License Version 2.0.
+// This product includes software developed at Datadog (https://www.datadoghq.com/).
+// Copyright 2016-present Datadog, Inc.
+
+//go:build linux
+
+// Package tags provides GPU-related host tags
+package tags
+
+import (
+	"github.com/DataDog/datadog-agent/pkg/util/log"
+	"github.com/NVIDIA/go-nvml/pkg/nvml"
+)
+
+var nvmlLibrary nvml.Interface
+
+// GetTags returns gpu_host:true if any NVIDIA GPUs are present, nil otherwise
+func GetTags() []string {
+	if nvmlLibrary == nil {
```

Comment:
This should only be called once, do we need to save the nvm lib in a global var ? If it's needed we need some king of mutex/sync to protect from race cond.